tabbycat/utils/tables.py

```diff
@@ -630,7 +630,7 @@ def add_motion_column(self, motions, show_order=False):
             } for motion in motions])
 
         motion_data = [{
-            'text': motion.reference if motion.reference else _('?'),
+            'text': motion.reference if motion.reference else _('??'),
```

Comment:
Hmm, I tried to test this too, but `motion.reference` is a mandatory field, lol. Where does it show up?